packages/vue/src/components/organisms/SfMegaMenu/_internal/SfMegaMenuColumn.js

```diff
@@ -0,0 +1,20 @@
+export default {
+  name: "SfMegaMenuColumn",
+  props: {
+    /**
+     * Column title
+     */
+    title: {
+      type: String,
+      default: ""
+    }
+  },
+  computed: {
+    isActive() {
+      return this.$parent.active.includes(this.title);
```

Comment:
be careful with this.$parent.At least do some error handling

Reply:
fixed: https://github.com/DivanteLtd/storefront-ui/pull/472/commits/9c97e5fdf97667972dfb156c80fdc780a0876faa